website/docs/dev_blender.md

```diff
@@ -0,0 +1,61 @@
+---
+id: dev_blender
+title: Blender integration
+sidebar_label: Blender integration
+toc_max_heading_level: 4
+---
+
+## Run python script at launch
+In case you need to execute a python script when Blender is started (aka [`-P`](https://docs.blender.org/manual/en/latest/advanced/command_line/arguments.html#python-options)), for example to programmatically modify a blender file for conformation, you can create an OpenPype hook as follows:
+
+```python
+from openpype.hosts.blender.hooks.pre_add_run_python_script_arg import AddPythonScriptToLaunchArgs
```

Comment:
You should not reference other hooks in hooks. I was testing this, and I noticed that `AddPythonScriptToLaunchArgs` hook was run twice, and it was because of this line.
Because this is just for the order, you could create a file `openpype/hosts/blender/constants.py` that keeps a constant that can be used by both `AddPythonScriptToLaunchArgs` and the hook to add the script.

Reply:
Fixed it